### PR TITLE
[djl-ui] Address security warning and upgrade to djl 0.6.0

### PR DESCRIPTION
**Pull request by frankfliu**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: visualization/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+package-lock.json
```
Comment (zachgk) [inline visualization/.gitignore:1]: This is meant to be checked in for reproducible builds. If you want to ignore it, delete the existing instances

**Comment by roywei** [comment]
shouldn't we also update the existing `package-lock.json` file for `websocket-extensions` ?

File: visualization/.gitignore (hunk 1 above)
Comment (frankfliu) [reply visualization/.gitignore:1]: I meant to delete this file

**Comment by frankfliu** [comment]
> shouldn't we also update the existing `package-lock.json` file for `websocket-extensions` ?

package-lock.json is generated file, we don't need checkin this file.

**Pull request merged**